sechub-storage-sharedvolume-spring/src/main/java/com/daimler/sechub/storage/sharevolume/spring/SharedVolumeJobStorage.java

```diff
@@ -0,0 +1,111 @@
+// SPDX-License-Identifier: MIT
+package com.daimler.sechub.storage.sharevolume.spring;
+
+import static java.util.Objects.*;
+
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.StandardCopyOption;
+import java.util.LinkedHashSet;
+import java.util.Set;
+import java.util.UUID;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.util.FileSystemUtils;
+import org.springframework.web.client.ResourceAccessException;
+
+import com.daimler.sechub.storage.core.JobStorage;
+import com.daimler.sechub.storage.core.StorageException;
+
+public class SharedVolumeJobStorage implements JobStorage {
+
+    private static final Logger LOG = LoggerFactory.getLogger(SharedVolumeJobStorage.class);
+    private String storagePath;
+    private UUID jobUUID;
+    private Path volumePath;
+
+    public SharedVolumeJobStorage(Path rootLocation, String storagePath, UUID jobUUID) {
+        requireNonNull(rootLocation, "rootlocation may not be null");
```

Comment:
```suggestion
        requireNonNull(rootLocation, "storage root location may not be null");
```

Reply:
the "requireNonNull" is the same as assertNotNull and message does always contain the name of the part which may not be null. So I do not change as described.

But:
- changed from `rootlocation` to  `rootLocation`
- changed from `projectId`to `storagePath` (missed while refactoring, done here now)